docs/content/preview/yugabyte-platform/manage-deployments/xcluster-replication/bidirectional-replication.md

```diff
@@ -0,0 +1,91 @@
+
+![Active-Active Multi-Master](/images/architecture/replication/active-active-deployment-new.png)
+
+YugabyteDB Anywhere detects an xCluster setup is for bidirectional if the tables that are being added to the replication are already part of a replication in the reverse direction (from A to B). In this scenario, YugabyteDB Anywhere skips the full copy for tables on universe B that are already in replication.
+
+{{<note title="Bidirectional xCluster deployment">}}
+Due to its operational complexity, bidirectional xCluster is not recommended for most use cases. If you are considering a bidirectional xCluster deployment, contact {{% support-general %}}.
+{{</note>}}
+
+## Limitations
+
+- Bidirectional replication can only be done using non-transactional replication. Transactional replication is not supported because transactional replication puts the target universe in read-only mode. For more information, see [Asynchronous replication modes](../../../../architecture/docdb-replication/async-replication/#asynchronous-replication-modes).
+- You can't use the YugabyteDB Anywhere UI to create two separate replication configurations for YSQL, each containing a subset of the database tables.
+
+For more information, refer to [Limitations](../../../../architecture/docdb-replication/async-replication/#limitations).
+
+## Prerequisites
+
+- Create two universes as described in [Prerequisites](../xcluster-replication-setup/#prerequisites).
+
+## Set up bidirectional replication
+
+To set up bidirectional replication from universe A to universe B, and vice versa on the same set of tables or databases, do the following:
+
+1. Make sure that there are no writes happening on universe B.
+1. Set up xCluster replication from A to B following the steps in [Deploy xCluster](../../../../deploy/multi-dc/async-replication/async-deployment/). A full copy of the selected tables or databases may be performed as part of this operation.
+1. Set up xCluster replication from universe B to A following the steps in [Deploy xCluster](../../../../deploy/multi-dc/async-replication/async-deployment/).
+
+    No full copy is performed, so it is important to have no writes on B during this setup, as these writes might not be replicated to universe A.
+
+For information on how to monitor and manage the replication configurations, refer to [Monitor replication](../xcluster-replication-setup/#monitor-replication).
+
+## Restart bidirectional replication
+
+Restarting bidirectional replication can cause unreplicated data on one of the universes to be lost, because a full copy is performed as part of the restart. Copying data can be done only if an xCluster configuration with reverse direction for a table does not exist, which means the reverse replication must first be deleted.
```

Comment:
```suggestion
In case of an extended network partition or replication is broken due to other reasons, it might be necessary to restart replication. However, restarting replication on individual directions of a bidirectional configuration does not achieve this goal.
```